src/test/prevector_tests.cpp

```diff
@@ -260,6 +276,14 @@ BOOST_AUTO_TEST_CASE(PrevectorTestInt)
             if (InsecureRandBits(5) == 18) {
                 test.move();
             }
+            if (InsecureRandBits(5) == 19) {
+                int num = 1 + (InsecureRandBits(4));
```

Comment:
Switch to unsigned? `InsecureRandBits` returns unsigned :-)

Reply:
@practicalswift Thank you. Oh, I see. ```unsigned int``` is better.
I have addressed it ( and rebased to be992701).